libmscore/harmony.cpp

```diff
@@ -634,17 +634,17 @@ const ChordDescription* Harmony::parseHarmony(const QString& ss, int* root, int*
       bool useLiteral = false;
       if (ss.endsWith(' '))
             useLiteral = true;
-      QString s = ss.simplified();
 
       if (_harmonyType == HarmonyType::ROMAN) {
-            _userName = s;
-            _textName = s;
+            _userName = ss;
+            _textName = ss;
```

Comment:
Does RNA need to avoid calling `simplified()` on its text? Do consequential spaces make sense there?

Reply:
Yes, it's good for RNA to not simplify, people may well want to embed multiple spaces to get various alignment effects.  That's why I changed this to use "ss", although that change is in the one commit I said I could remove for now if necessary.

Reply:
and btw I have my branch update locally to remove that commit, it works as expected for me, so I could force push it if we decide to skip the special characters improvements

Reply:
Oh, yeah, also - I anticipate supporting mutli-line RNA at some point, that was my original reason to get rid of the simplified.  I had multiline partially working, but underlying issues in handlling of multiline text in TextBase made me stop for now.